docs/source/user_manual/quickstart.rst

```diff
@@ -1,267 +0,0 @@
-`Github <https://github.com/enthought/chaco>`_ or download a source
-distribution from PyPi. If you don't want a local copy of the Chaco codebase,
-you can browse the  
-`examples on GitHub <https://github.com/enthought/chaco/tree/master/examples>`_.
-
-In the :file:`examples/demo/` directory, examples are classified by themes and
-located in separate directories.  Almost all of the Chaco examples are
-standalone files that can be run individually. We will first show how to
-execute them from the command line, and then we will show how to run Chaco in
-an interactive way from IPython. This "shell" mode will be more familiar to
-Matplotlib or MATLAB users.
-
-.. note::
-   Some of these examples can be visualized in our
-   :ref:`annotated examples <examples>`.
-
-
-First plots from the command line
----------------------------------
-
-From the :file:`examples/demo` directory, run the ``simple_line`` example::
-
-    python simple_line.py
-
-This opens a plot of several Bessel functions with a legend.
-
-.. image:: images/simple_line.png
-
-You can interact with the plot in several ways:
-
-.. Ctrl-Left and Ctrl-Right don't work in OS X?
-
-* To pan the plot, hold down the left mouse button inside the plot area (but
-  not on the legend) and drag the mouse.
-
-* To zoom the plot:
-
-  * Mouse wheel: scroll up to zoom in, and scroll down to zoom out (or the
-    reverse you're on a version of OS X with 'natural scrolling').
-
-  * Zoom box: Press :kbd:`z`, and then draw a box region to zoom in on.
-    (There is no box-based zoom out.) Press :kbd:`Ctrl-Left` and
-    :kbd:`Ctrl-Right` to go back and forward in your zoom box history.
-
-  * Drag: hold down the right mouse button and drag the mouse up or down. Up
-    zooms in, and down zooms out.
-
-  * For any of the above, press :kbd:`Escape` to reset the zoom to the
-    original view.
-
-* To move the legend, hold down the right mouse button inside the legend and
-  drag it around. Note that you can move the legend outside of the plot area.
-
-* To exit the plot, click the "close window" button on the window frame or (on
-  Mac) choose the Quit option on the Python menu.  Alternatively, can you press
-  :kbd:`Ctrl-C` in the terminal.
-
-You can run most of the examples in the the :file:`examples/demo/basic/`
-directory and the :file:`examples/demo/shell/` directory.  The
-:file:`examples/demo/advanced/` directory has some examples that require
-additional data or packages. In particular,
-
-* :file:`spectrum.py` requires that you have PyAudio installed and a working
-  microphone.
-
-* :file:`data_cube.py` needs to download about 7.3mb of data from the Internet
-  the first time it is executed, so you must have a working Internet
-  connection. Once the data is downloaded, you can save it so you can run the
-  example offline in the future.
-
-For detailed information about each built-in example, see the :ref:`examples`
-section.
-
-
-First plots from IPython
-------------------------
```

Comment:
I thought we had discussed this while we went through the docs.  The other sections were all redundant from the tutorial / I had "kill quickstart" written in my notes from our discussion.

I had this removal in a separate PR specifically because it wasn't only motivated by the removal of `chaco.shell`

Reply:
sorry. you're right. i was mistaking this page for something else. approving.